### PR TITLE
Jenkins Happy Notification

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Adds a notification to Slack when Jenkins recovers after a failure.

Inspired by https://stackoverflow.com/a/45777893

### Demo

image

### CODE REVIEW

**Comment by rajadain** [comment]
@azavea-bot rebuild

File: Jenkinsfile [hunk 1]
```diff
@@ -66,6 +66,17 @@ node {
...
 		}
+
+        stage('notify') {
+            if (currentBuild.currentResult == 'SUCCESS' && currentBuild.previousBuild?.result != 'SUCCESS') {
+                def slackMessage = ":jenkins: *Open Apparel Registry (${env.BRANCH_NAME}) #${env.BUILD_NUMBER}*"
+                if (env.CHANGE_TITLE) {
+                    slackMessage += "\n${env.CHANGE_TITLE} - ${env.CHANGE_AUTHOR}"
+                }
+                slackMessage += "\n<${env.BUILD_URL}|View Build>"
+                slackSend  channel: '#oar', color: '#009452', message: slackMessage
```
Comment (rbreslow) [inline Jenkinsfile]:
```suggestion
                slackSend channel: '#oar', color: 'good', message: slackMessage
```

**Comment by rajadain** [comment]
Yes I'm aware of the Slack configuration available in the Jenkins UI, but all my searching (in both Pipeline and Groovy implementations) there isn't a native alternative for the Jenkinsfile. Maybe this pattern will end up being adopted in other projects until such functionality is available out of the box.

**Comment by rajadain** [comment]
Thanks for pointing out the `good` magic word. I'd tried `success` before to no avail. Adopted. ✅

**Pull request merged**